src/decs.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'react-country-flag';
```

Comment:
The common practice is to keep separate `.d.ts` files per package. So this would go in `src/react-country-flag.d.ts`. Also someone has already [written a declaration file](https://github.com/danalloway/react-country-flag/issues/16#issuecomment-665846925) containing some types, so you should use that for extra safety.